src/openms/source/FORMAT/HANDLERS/MzMLSpectrumDecoder.cpp

```diff
@@ -306,6 +306,7 @@ namespace OpenMS
 
     if (data.size() > 2) 
     {
+      spectrum.sortByPosition();
```

Comment:
is this correct regarding potential data arrays? is this from the other PR?

Reply:
This is vestigial from another branch. I'm gonna work to disentangle this one.